### DISCUSSION TITLE
[Help Request] Error Cloning UnrealEngine's repository.

### DISCUSSION THREAD

**Discussion by ahmedgaberntust** [carla-simulator/carla#8140] [Q&A]
Hello everyone, I've been trying to clone UnrealEngine's modified version's repository for CARLA (https://github.com/CarlaUnreal/UnrealEngine). I've been following the installation steps, one by one. However, i keep encountering the following error when i try to clone the repository:
```
C:\>git clone --depth 1 -b carla https://github.com/CarlaUnreal/UnrealEngine.git
Cloning into 'UnrealEngine'...
remote: Repository not found.
fatal: repository 'https://github.com/CarlaUnreal/UnrealEngine.git/' not found
```
I have tried other methods, for example downloading Github desktop and try to clone the repository from there but I've encountered the same error (Repository not found).

**Comment by Blyron** [comment]
Join Epic games organization in github and try again
